client/state/gutenberg-fse-settings/selectors/is-core-fse-eligible.ts

```diff
@@ -0,0 +1,14 @@
+import { queryCache } from 'calypso/state/query-cache';
+import { makeQueryKey, SuccessResponse, GutenbergFSESettings } from '../init';
+
+export const isCoreFSEEligible = ( siteId: string ): boolean => {
+	const cached = queryCache.find<
+		SuccessResponse< GutenbergFSESettings >,
+		unknown,
+		GutenbergFSESettings
+	>( makeQueryKey( siteId ) );
```

Comment:
So we can query cached data inside the selectors.

Reply:
Can be done on the `queryClient` object:
```js
queryClient.getQueryData( makeQueryKey( siteId ) )?.is_core_fse_eligible ?? false;
```
The `getQueryData` method works on the internal `QueryCache` instance. The cache object can remain invisible to Calypso code.